app/src/main/java/fr/groggy/racecontrol/tv/ui/season/browse/SeasonBrowseFragment.kt

```diff
@@ -49,6 +69,46 @@ class SeasonBrowseFragment : BrowseSupportFragment(), OnItemViewClickedListener
         adapter = eventsAdapter
     }
 
+    /**
+     * Function that will load a [Session.largePictureUrl] as a background image when a card is selected
+     */
+    private fun initializeBackground() {
+        val backgroundManager = BackgroundManager.getInstance(activity).apply {
+            attach(activity?.window)
+        }
+
+        val metrics = DisplayMetrics()
+        requireActivity().windowManager.defaultDisplay.getMetrics(metrics)
+
+        onItemViewSelectedListener = OnItemViewSelectedListener { _, item, _, _ ->
+            if (item is Session) {
+                viewLifecycleOwner.lifecycleScope.launch {
+                    delay(BACKGROUND_UPDATE_DELAY) // A small delay so that the loading of the image and transition is more smooth
+
+                    Glide.with(requireActivity())
+                        .load(item.largePictureUrl)
+                        .centerCrop()
```

Comment:
Hm, can you test this in a 720p tv emulator or so?

What does this centerCrop does there?

Is it really needed?

Reply:
I can no longer test it since my account has ended and i wont be renewing it untill the 2022 season starts.
And as for screenshoots the most recent ones that i have are in here -~>  #37

Reply:
:D Same for me, I am merging it then will try out.